db/background_mgr_attachment_migration.go

```diff
@@ -76,11 +78,15 @@ func (a *AttachmentMigrationManager) Init(ctx context.Context, options map[strin
 		if statusDoc.State == BackgroundProcessStateCompleted || err != nil || reset {
 			return newRunInit()
 		}
-		a.MigrationID = statusDoc.MigrationID
-		a.SetStatus(statusDoc.DocsChanged, statusDoc.DocsProcessed)
-		a.SetCollectionIDs(statusDoc.CollectionIDs)
+		a.lock.Lock()
```

Comment:
Since the stats are all atomics I'm assuming the lock is for migrationID and collectionIDs.  We don't use locking anywhere else when accessing migrationID (including this function), and have an existing function (SetCollectionIDs) to manage locking for CollectionIds, so the lock/defer unlock approach doesn't feel like a good fit.

Reply:
I removed this code. I do not think it is right but it is outside the scope of this problem.